bcipy/acquisition/client.py

```diff
@@ -87,6 +90,9 @@ def __init__(self,
         self._max_wait = 0.1  # for process loop
 
         self.marker_writer = NullMarkerWriter()
+
+        # column in the acquisition data used to calculate offset.
+        self.offset_column = OFFSET_COLUMN_NAME
```

Comment:
I think it might be more meaningful to call it a trigger column and require that it's present. We're going to refactor the way we handle offset correction in a future ticket.

Reply:
Yeah, I went back and forth on this one. I'll go ahead and rename it to trigger_column for clarity.